schema/gks-core/gks-core-source.yaml

```diff
@@ -188,26 +188,40 @@ $defs:
     description: >-
       A structured representation of a code for a defined concept in a terminology or code system.
     properties:
+      code:
+        $ref: '#/$defs/code'
+        description: >-
+          A symbol uniquely identifying the concept, as in a syntax defined by the code system.
+          CURIE format is preferred where possible (e.g. 'SO:0000704' is the CURIE form of the
+          Sequence Ontology code for 'gene'). See https://www.w3.org/TR/curie/.
       label:
         type: string
         description: The human-readable name for the coded concept, as defined by the code system.
       system:
         type: string
         description: >-
           The terminology/code system that defined the code. May be reported as a free-text name
-          (e.g. 'Sequence Ontology'), but it is preferable to provide a uri/url for the system.
-          When the 'code' is reported as a CURIE, the 'system' should be reported as the uri that
-          the CURIE's prefix expands to (e.g. 'http://purl.obofoundry.org/so.owl/' for the
-          Sequence Ontology).
+          (e.g. 'Sequence Ontology'), but it is preferrable to provide an IRI or URL that resolves to 
+          a specification of the system, or its home page. e.g. for the Sequence Ontology, 
+          'http://purl.obolibrary.org/so.owl' resolves to an owl rendering of the ontology, and
+          'http://www.sequenceontology.org/' resolves to its home page.
       systemVersion:
         type: string
         description: Version of the terminology or code system that provided the code.
-      code:
-        $ref: '#/$defs/code'
+      systemPrefix: 
```

Comment:
I think I'm a little confused on the purpose of `systemPrefix` if `code` is preferred to be a CURIE (`systemPrefix:identifier`).

Reply:
If we want to include `systemPrefix` maybe we just have `code` be the identifier?

Reply:
`systemPrefix` and `systemPrefixExpansion` would just allow data providers to be explicit about what the prefix is, and what it expands to in order to generate a resolvable url - if they wanted to.  This kind of info is often captured in prefix mapping sections of schema in the linkML / linked data world.  Not sure we need it here right now - but in combination with the other proposed changes it lets a data provider be very clear about the system, the code from that system, and how to resolve this code to find more info about it (in cases where the code is a CURIE with a url prefix that lets it resolve). Whether this is useful/necessary to support is another question.